tutor/config.py

```diff
@@ -319,3 +328,21 @@ def _update_enabled_plugins_on_unload(_plugin: str, _root: str, config: Config)
     Note that this action must be performed after the plugin has been unloaded, hence the low priority.
     """
     save_enabled_plugins(config)
+
+
+@hooks.Actions.CONFIG_LOADED.add()
+def _check_preview_lms_host(config: Config) -> None:
+    """
+    This will check if the PREVIEW_LMS_HOST is a subdomain of LMS_HOST.
+    if not, prints a warning to notify the user.
+    """
+
+    lms_host = str(config.get("LMS_HOST"))
```

Comment:
`lms_host = config.get("LMS_HOST", "")`

Reply:
Actually, you should fix type tests by writing:

```
from tutor.types import get_typed
lms_host = get_typed(config, "LMS_HOST", str, "")
```

Same for `PREVIEW_LMS_HOST` below.